paddle/fluid/platform/gpu_info.cc

```diff
@@ -641,6 +646,30 @@ class RecordedCudaMallocHelper {
 
   uint64_t LimitSize() const { return limit_size_; }
 
+#ifdef PADDLE_WITH_CUDA
+#if CUDA_VERSION >= 10020
+  CUresult cuMemCreate(CUmemGenericAllocationHandle *handle, size_t size,
```

Comment:
Better name the member function like other functions, for example, "CreateMem". (Start with upper case)

Reply:
done，thx！